services/tables/src/test/resources/cluster-test-properties.yaml

```diff
@@ -2,6 +2,21 @@ cluster:
   name: "TestCluster"
   storage:
     root-path: "/tmp/unittest"
+  storages:
+    default-type: "hdfs"
+    types:
+      hdfs:
+        scheme: "hdfs"
+        rootpath: "/tmp/unittest"
+        endpoint: "hdfs://localhost:9000"
+        parameters:
+          key1: value1
+      s3:
+        scheme: "s3"
+        rootpath: "tmpbucket"
+        endpoint: "http://localhost:9000"
+        parameters:
+          key2: value2
```

Comment:
why aren't we not having a `local` segment here instead?

Reply:
This `cluster.yaml` is just for testing. 

As for introducing a local or not, I wanted to keep it optional for backward-compatibility with tables-test-fixtures. ie. If no storages are provided, Local would be used by default.